frontend/lib/services/authentication/authentication_service.dart

```diff
@@ -34,11 +34,6 @@ class AuthenticationService {
     return _saveUser(result);
   }
 
-  Future<Result<void>> signUpWithGoogle() async {
-    final result = await _firebaseAuthenticator.signUpWithGoogle();
-    return await _saveUser(result).flatMapFuture((user) => _authenticationApi.addUser(AddUserRequest(email: user.email, userId: user.id)));
-  }
-
```

Comment:
For this service, we should keep `signUpWithGoogle`, because we should tell the backend that a new user is created

Reply:
I add this again and assigned into RegisterWithGoogleRequest event